solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/ZkStateReader.java

```diff
@@ -88,6 +88,9 @@ public class ZkStateReader implements SolrCloseable {
   public static final String CORE_NAME_PROP = "core";
 
   public static final String COLLECTION_PROP = "collection";
+  public static final String INCLUDE_ALL = "includeAll";
+  public static final String LIVENODES_PROP = "liveNodes";
+  public static final String CLUSTER_PROP = "clusterProperties";
```

Comment:
No; please don't add constants to ZkStateReader that are really HTTP parameter level constants.  Instead, see CollectionAdminParams if you must... but for this PR, these constants are really specific to a particular request, so I suggest just leaving them as constants in ClusterStatus.java

I'm aware some older constants here are used for multiple purposes but that's a bad practice.  The constants here _should_ only be used for objects in ZK.

Reply:
Addressed in latest commit

Reply:
they are still here.

Reply:
Should be gone now.